device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O28/hwsku.json

```diff
@@ -113,20 +113,28 @@
             "subport": "1"
         },
         "Ethernet224": {
-	     "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
-            "subport": "1"
+            "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
+            "subport": "1",
+            "autoneg": "on",
+            "role": "Dpc"
         },
         "Ethernet232": {
-	     "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
-            "subport": "1"
+            "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
+            "subport": "1",
+            "autoneg": "on",
+            "role": "Dpc"
         },
         "Ethernet240": {
-	     "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
-            "subport": "1"
+            "default_brkout_mode": "1x200G[400G,100G,50G,40G,25G,10G,1G]",
```

Comment:
why breakout?

Reply:
Nothing was changed about breakout. Only new fields were added

Reply:
breakout is not applicable for these ports

Reply:
This mode `1x200G[400G,100G,50G,40G,25G,10G,1G]` would mean that the internal ports would be of 200G and will be using all the 8 lanes. No breakout is performed automatically here because of having this mode.

Reply:
Even we don't see any speed change. I think port would be of 400G (not 200G), right?

Reply:
Resolved offline